src/Workspaces/CSharp/Portable/Classification/Worker.cs

```diff
@@ -104,6 +106,10 @@ private void ClassifyToken(SyntaxToken token)
                     {
                         AddClassification(span, ClassificationTypeNames.StaticSymbol);
                     }
+                    else if (token.IsKind(SyntaxKind.Utf8StringLiteralToken, SyntaxKind.Utf8SingleLineRawStringLiteralToken, SyntaxKind.Utf8MultiLineRawStringLiteralToken) && token.Text.EndsWith("u8", StringComparison.OrdinalIgnoreCase))
+                    {
+                        AddClassification(new TextSpan(token.Span.End - 2, 2), ClassificationTypeNames.Keyword);
+                    }
```

Comment:
what about utf8 interpolated strings?  Or is that not a thing?

Reply:
It's not a thing.